installing/installing_aws/installing-aws-account.adoc

```diff
@@ -22,6 +22,8 @@ include::modules/installation-aws-iam-user.adoc[leveloffset=+1]
 
 * See xref:../../installing/installing_aws/manually-creating-iam.adoc#manually-creating-iam-aws[Manually creating IAM for AWS] for steps to set the Cloud Credential Operator (CCO) to manual mode prior to installation. Use this mode in environments where the cloud identity and access management (IAM) APIs are not reachable, or if you prefer not to store an administrator-level credential secret in the cluster `kube-system` project.
 
+include::modules/manually-removing-cloud-creds.adoc[leveloffset=+1]
```

Comment:
Same as in https://github.com/openshift/openshift-docs/pull/32152 - I think the change looks ok, but it should go in the post-install content instead.

Reply:
It might also be worth adding the xrefs to the new, post installation location to the relevant assemblies, like in the 4.7+ PR.